### ISSUE TITLE
Adding more contrast/color to the sidebar icons.

### ISSUE THREAD

**Issue by youbzh** [getstation/desktop-app#239]
I'm starting to use Station and really love it, thanks!
However I find the sidebar very unusable at the moment. The contrast for icons that are not active makes them difficult to differentiate from another:
<img width="45" alt="image" src="https://github.com/getstation/desktop-app/assets/95705722/8e78eac0-3cec-4585-a343-ac76d81d2d99">
I'd suggest two options:
• making the contrast higher : darker blue background and brighter/whiter symbol
• or having colored icons for each app, and making the active app's icon bigger, or circled with a thicker white line to differentiate it

Thanks for your attention

**Comment by razzamatazm** [comment]
Bumping this. Sidebar shouldn't be distracting, but should be usable without a magnifying glass. Icons are iconic for a reason. I recognize logos from a mile away!

Color would make this super helpful!

**Comment by 2ge** [comment]
hi, I am moving out from Ferdium, Station looks like good alternative, but this is exactly issue what I have - I need to have color icons. Now I dont know which app is which one, with color icons I can be 2-3 times faster between switching applications. Please add it.

**Comment by Vegals** [comment]
Yeah, bumping this. Station is great but the side bar is so tiny, and bummer I can't choose my own colours for the background and icon styling. Probably not relevant but the minimise, close, and full screen buttons on Mac are smaller than all other Mac apps.

**Comment by Hellmark** [comment]
It would be nice to allow the sidebar to be themeable. I use blue sparingly, because certain blue lights mess with my eyes. For me, this makes the sidebar hard to see. Only sort of theming I've seen is for icon color when making a custom addition, but that doesn't seem to be something you can change after you initially set it.
